Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackAndGenerateCode.cpp

```diff
@@ -149,12 +149,17 @@ static ALWAYS_INLINE CCallHelpers::Address callFrameAddr(CCallHelpers& jit, intp
         return CCallHelpers::Address(GPRInfo::callFrameRegister, offsetFromFP);
     }
 
+    if (isARM()) {
+        // For now, we can just solve this in the macro assembler ...
+        return CCallHelpers::Address(GPRInfo::callFrameRegister, offsetFromFP);
+    }
+
     auto addr = Arg::addr(Air::Tmp(GPRInfo::callFrameRegister), offsetFromFP);
-    if (addr.isValidForm(Width64))
+    if (addr.isValidForm(pointerWidth()))
```

Comment:
In ARM64_32, pointer-width is 32bit while register is 64bit. We should have notion of `registerWidth`.